spec/features/state_file/complete_intake_spec.rb

```diff
@@ -320,6 +320,10 @@
   end
 
   context "NC", :flow_explorer_screenshot, js: true do
+    before do
+      allow_any_instance_of(Efile::Nc::D400Calculator).to receive(:refund_or_owed_amount).and_return 1000
```

Comment:
is this because the fixture "transfer Nick" doesn't have `refund_or_owed_amount`?

Reply:
yes! that work is in a story that is currently blocked (i started adding this test when i picked up that story, before i realized it was blocked)